packages/orchestration/src/orchestration-types.ts

```diff
@@ -56,3 +58,42 @@ export interface OrchestrationModuleConfig {
    */
   grounding?: GroundingModuleConfig;
 }
+
+/**
+ * Represents a filter configuration for the Document Grounding Service.
+ *
+ */
+export type DocumentGroundingServiceFilter = Pick<
+  DocumentGroundingFilter,
+  | 'id'
+  | 'search_config'
+  | 'data_repositories'
+  | 'data_repository_metadata'
+  | 'document_metadata'
+  | 'chunk_metadata'
+> & {
+  /**
+   * Defines the type of data repository.
+   * If not set, the default value is 'vector'.
+   */
+  data_repository_type?: DataRepositoryType;
+};
+
+/**
+ * Represents the configuration for the Document Grounding Service.
+ */
+export interface DocumentGroundingServiceConfig {
+  /**
+   * Define the filters to apply during the grounding process.
+   */
+  filters?: DocumentGroundingServiceFilter[];
+  /**
+   * Contains the input parameters used for grounding input questions.
+   */
+  input_params: string[];
+  /**
+   * Parameter name used for grounding output.
+   * @example "groundingOutput"
```

Comment:
[q] Not sure if this example helps a lot. Makes only sense to us probably since we used `groundingOutput`.